### ISSUE TITLE
Minecraft Crash when Energy Conduit connect with Capacitor Bank

### ISSUE THREAD

**Issue by csum367** [CrazyPants/EnderIO#1584]
When i was testing something in creative i was trying out the new Capacitor bank but when i placed a conduit next to it, my minecraft crash. Then the world is not acessible anymore unless i use mc edit to delete the block. I tested the problem with all conduit and all capacitor bank and none of them work.

Crash Report:

--- Minecraft Crash Report ----
// I just don't know what went wrong :(

Time: 1/5/15 2:35 PM
Description: Exception in server tick loop

java.lang.IncompatibleClassChangeError: Class crazypants.enderio.machine.capbank.TileCapBank does not implement the requested interface cofh.api.energy.IEnergyReceiver
	at crazypants.enderio.power.EnergyReceiverPI.canConduitConnect(EnergyReceiverPI.java:22)
	at crazypants.enderio.conduit.power.PowerConduit.canConnectToExternal(PowerConduit.java:424)
	at crazypants.enderio.conduit.AbstractConduit.onAddedToBundle(AbstractConduit.java:471)
	at crazypants.enderio.conduit.TileConduitBundle.addConduit(TileConduitBundle.java:337)
	at crazypants.enderio.conduit.AbstractItemConduit.func_77648_a(AbstractItemConduit.java:65)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:454)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 556378640 bytes (530 MB) / 1507180544 bytes (1437 MB) up to 2093809664 bytes (1996 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn512M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 15, tallocated: 95
	FML: MCP v9.05 FML v7.10.85.1230 Minecraft Forge 10.13.2.1230 71 mods loaded, 71 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	FML{7.10.85.1230} [Forge Mod Loader] (forge-1.7.10-10.13.2.1230.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Forge{10.13.2.1230} [Minecraft Forge] (forge-1.7.10-10.13.2.1230.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	appliedenergistics2-core{rv1-stable-1} [AppliedEnergistics2 Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CodeChickenCore{1.0.4.29} [CodeChicken Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	<CoFH ASM>{000} [CoFH ASM Data Initialization] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	InfiniBows{1.3.0 build 20} [InfiniBows] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NotEnoughItems{1.0.4.83} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.4.83-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThaumicTinkerer-preloader{0.1} [Thaumic Tinkerer Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenModsCore{@VERSION@} [OpenModsCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	debug{1.0} [debug] (denseores-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	appliedenergistics2{rv1-stable-1} [Applied Energistics 2] (appliedenergistics2-rv1-stable-1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CoFHCore{1.7.10R3.0.0B6} [CoFH Core] (CoFHCore-[1.7.10]3.0.0B6-32.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	autopackager{1.5.2} [AutoPackager] (autopackager-1.5.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Backpack{2.0.1} [Backpack] (backpack-2.0.1-1.7.x.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BiblioCraft{1.8.1} [BiblioCraft] (BiblioCraft[v1.8.1][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThermalFoundation{1.7.10R1.0.0B3} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.0.0B3-8.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThermalExpansion{1.7.10R4.0.0B5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.0.0B5-13.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BigReactors{0.4.1A2} [Big Reactors] (BigReactors-0.4.1A2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	AWWayofTime{v1.3.0a} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.0a-1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Thaumcraft{4.2.2.1} [Thaumcraft] (Thaumcraft-1.7.10-4.2.2.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Botania{r1.3-149} [Botania] (Botania r1.3-149.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Core{6.2.6} [BuildCraft] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Transport{6.2.6} [BC Transport] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Silicon{6.2.6} [BC Silicon] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Builders{6.2.6} [BC Builders] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Energy{6.2.6} [BC Energy] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Factory{6.2.6} [BC Factory] (buildcraft-6.2.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CarpentersBlocks{3.3.4.2} [Carpenter's Blocks] (Carpenter's Blocks v3.3.4.2 - MC 1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	chisel{1.5.7} [Chisel] (Chisel-1.7.10-1.5.7.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	IC2{2.2.648-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.648-experimental.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CompactSolars{4.4.37.314} [Compact Solar Arrays] (CompactSolars-1.7.10-4.4.37.314-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	eplus{3.0.2-d} [Enchanting Plus] (EnchantingPlus-1.7.10-3.0.2-d.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	EnderIO{1.7.10-2.2.6.317} [Ender IO] (EnderIO-1.7.10-2.2.6.317.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	EnderStorage{1.4.5.27} [EnderStorage] (EnderStorage-1.7.10-1.4.5.27-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Waila{1.5.8} [Waila] (Waila-1.5.8a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	extracells{2.1.12} [ExtraCells] (ExtraCells-1.7.10-2.1.12b12.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMultipart{1.1.0.310} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.0.310-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ExtraUtilities{1.2.1} [Extra Utilities] (extrautilities-1.2.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	inventorytweaks{1.59-dev-156-af3bc68} [Inventory Tweaks] (InventoryTweaks-1.59-dev-156.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	IronChest{6.0.60.741} [Iron Chest] (ironchest-1.7.10-6.0.60.741-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	JABBA{1.2.0a} [JABBA] (Jabba-1.2.0a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	journeymap{5.0.0RC2} [JourneyMap] (JourneyMap5.0.0RC2_Unlimited_MC1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Mystcraft{0.11.0.00} [Mystcraft] (mystcraft-1.7.10-0.11.0.00.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons{1.12.2.9} [NEI Addons] (neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|Botany{1.12.2.9} [NEI Addons: Botany] (neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|Forestry{1.12.2.9} [NEI Addons: Forestry] (neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|CraftingTables{1.12.2.9} [NEI Addons: Crafting Tables] (neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|ExNihilo{1.12.2.9} [NEI Addons: Ex Nihilo] (neiaddons-mc1710-1.12.2.9.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	neiintegration{1.0.0} [NEI Integration] (NEIIntegration-MC1.7.10-1.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIPlugins{1.1.0.11-unofficial} [NEI Plugins Unofficial] (NEIPlugins-MC1.7.10-1.1.0.11-unofficial.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NotEnoughCodecs{0.3} [NotEnoughCodecs] (NotEnoughCodecs-1.7.10-0.3-snapshot-11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenMods{0.6} [OpenMods] (OpenModsLib-1.7.10-0.6-snapshot-224.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenBlocks{1.3} [OpenBlocks] (OpenBlocks-1.7.10-1.3-snapshot-433.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Core{4.5.1.51} [ProjectRed] (ProjectRed-Base-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Transmission{4.5.1.51} [ProjectRed-Transmission] (ProjectRed-Integration-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Exploration{4.5.1.51} [ProjectRed-Exploration] (ProjectRed-World-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	TConstruct{1.7.10-1.8.1.build815} [Tinkers' Construct] (TConstruct-1.7.10-1.8.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Compatibility{4.5.1.51} [ProjectRed-Compatibility] (ProjectRed-Compat-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Integration{4.5.1.51} [ProjectRed-Integration] (ProjectRed-Integration-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Illumination{4.5.1.51} [ProjectRed-Illumination] (ProjectRed-Lighting-1.7.10-4.5.1.51.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Railcraft{9.4.0.0} [Railcraft] (Railcraft_1.7.10-9.4.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	StevesCarts{2.0.0.b18} [Steve's Carts 2] (StevesCarts2.0.0.b18.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	StevesFactoryManager{A93} [Steve's Factory Manager] (StevesFactoryManagerA93.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThaumicTinkerer{unspecified} [Thaumic Tinkerer] (ThaumicTinkerer-2.5-1.7.10-162.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	WailaHarvestability{1.1.1} [Waila Harvestability] (WailaHarvestability-mc1.7.x-1.1.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	McMultipart{1.1.0.310} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.0.310-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	denseores{1.0} [Dense Ores] (denseores-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMicroblock{1.1.0.310} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.0.310-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	AE2 Version: stable rv1-stable-1 for Forge 10.13.0.1187
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	AE2 Integration: IC2:ON, RotaryCraft:OFF, RC:ON, BC:ON, MJ6:OFF, MJ5:OFF, RF:ON, RFItem:ON, MFR:OFF, DSU:ON, FZ:OFF, FMP:ON, RB:OFF, CLApi:OFF, Waila:ON, InvTweaks:ON, NEI:ON, CraftGuide:OFF, Mekanism:OFF, ImmibisMicroblocks:OFF, BetterStorage:OFF
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['csum367'/282, l='EnderIO Test', x=129.71, y=64.00, z=199.35]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by SpyroForever** [comment]
YOU'RE not alone:)

---- Minecraft Crash Report ----
// Oh - I know what I did wrong!

Time: 06.01.15 19:04
Description: Exception in server tick loop

java.lang.IncompatibleClassChangeError: Class crazypants.enderio.machine.power.TileCapacitorBank does not implement the requested interface cofh.api.energy.IEnergyReceiver
	at crazypants.enderio.power.EnergyReceiverPI.canConduitConnect(EnergyReceiverPI.java:22)
	at crazypants.enderio.conduit.power.PowerConduit.canConnectToExternal(PowerConduit.java:424)
	at crazypants.enderio.conduit.AbstractConduit.onAddedToBundle(AbstractConduit.java:471)
	at crazypants.enderio.conduit.TileConduitBundle.addConduit(TileConduitBundle.java:337)
	at crazypants.enderio.conduit.AbstractItemConduit.func_77648_a(AbstractItemConduit.java:65)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:454)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

**Comment by melekaiah** [comment]
this is not just tilecapacitor bank its also the solar generator from extra utilities. 

java.lang.IncompatibleClassChangeError: Class com.rwtema.extrautils.tileentity.generator.TileEntityGeneratorSolar does not implement the requested interface cofh.api.energy.IEnergyReceiver
	at crazypants.enderio.power.EnergyReceiverPI.canConduitConnect(EnergyReceiverPI.java:22)
	at crazypants.enderio.conduit.power.PowerConduit.canConnectToExternal(PowerConduit.java:424)
	at crazypants.enderio.conduit.AbstractConduit.updateConnections(AbstractConduit.java:409)
	at crazypants.enderio.conduit.AbstractConduit.updateEntity(AbstractConduit.java:393)
	at crazypants.enderio.conduit.TileConduitBundle.func_145845_h(TileConduitBundle.java:216)
	at net.minecraft.world.World.func_72939_s(World.java:1939)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at crazypants.enderio.power.EnergyReceiverPI.canConduitConnect(EnergyReceiverPI.java:22)
	at crazypants.enderio.conduit.power.PowerConduit.canConnectToExternal(PowerConduit.java:424)
	at crazypants.enderio.conduit.AbstractConduit.updateConnections(AbstractConduit.java:409)
	at crazypants.enderio.conduit.AbstractConduit.updateEntity(AbstractConduit.java:393)
	at crazypants.enderio.conduit.TileConduitBundle.func_145845_h(TileConduitBundle.java:216)

-- Block entity being ticked --
Details:
	Name: blockConduitBundleTileEntity // crazypants.enderio.conduit.TileConduitBundle
	Block type: ID #1244 (tile.blockConduitBundle // crazypants.enderio.conduit.BlockConduitBundle)
	Block data value: 0 / 0x0 / 0b0000
	Block location: World: (-151,53,-2248), Chunk: (at 9,3,8 in -10,-141; contains blocks -160,0,-2256 to -145,255,-2241), Region: (-1,-5; contains chunks -32,-160 to -1,-129, blocks -512,0,-2560 to -1,255,-2049)
	Actual block type: ID #1244 (tile.blockConduitBundle // crazypants.enderio.conduit.BlockConduitBundle)
	Actual block data value: 0 / 0x0 / 0b0000
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1939)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)

**Comment by iTitus** [comment]
Please, please, please, please read the issues and changelogs before posting issues here.
Please.

**Comment by SpyroForever** [comment]
I need help!!!!

**Comment by iTitus** [comment]
I offered you help, use it.

**Comment by melekaiah** [comment]
Changelogs are where? and how do these apply to this issue? using latest version of mod. this crashes server when used with many other mods. hence why its being reported here by a few people who may not know better.

**Comment by iTitus** [comment]
See #1415 #1422 #1425 #1467 #1476 #1517

**Comment by iTitus** [comment]
Or see #1542 #1529 #1511 #1498 #1464 #1438 #1419 #1406 #1376
TL;DR: http://minecraft.curseforge.com/mc-mods/64578-ender-io/files/2222045

**Comment by iTitus** [comment]
Even more TL;DR:
Important: EIO now requires the latest version of the RF API. This change means EIO conduits will work with the latest release of TE and MFR, but it will also mean you may get errors / crashes if you use mods that ship an old version of the API. If you are getting java.lang.IncompatibleClassChangeError errors this means an old version of the RF is being loaded. This can resolved by either removing the cofh/api/energy folder from the out of date mods jars, or including the latest cofh core (as is the the owner of the RF api its version should always be loaded). Do not report these errors on the EIO issue tracker, but please do report them to the owners of the mods including the old API.
